runtime/runtime-domain-service/ballerina/build.gradle

```diff
@@ -16,12 +16,37 @@
 // under the License.
 //
 
-plugins {
-    id 'net.researchgate.release' version '3.0.2'
-}
+import org.apache.tools.ant.taskdefs.condition.Os
 
 apply from: "$projectDir/../../../common-gradle-scripts/ballerina.gradle"
-
-tasks.named('build').configure {
+task build{
+    mustRunAfter 'updateTomlFiles'
     dependsOn 'bal_build'
```

Comment:
Should `bal_build` run after `updateTomlFiles` or before `updateTomlFiles`? because this block doesn't verify the order. If `updateTomlFiles` and `bal_build` are independent tasks this block is ok.